manuskript.spec

```diff
@@ -38,3 +49,11 @@ coll = COLLECT(exe,
                strip=False,
                upx=True,
                name='manuskript')
+
+# also build an app bundle on macOS
+if platform == 'mac':
+    app = BUNDLE(coll,
+        name='Manuskript.app',
+        icon=os.path.join(SPECPATH, 'icons/Manuskript/manuskript.ico'),
```

Comment:
You probably want the Mac-specific icon here. (But you took care of that in another commit.)

I know @gedakc is a fan of having a single commit with the final product before merging the pull request, so you may want to look into consolidating these commits.

Reply:
For a change such as this a single commit is preferred.  As it currently stands the set of commits add a `manuskript-macos.spec` file and then effectively delete the file and merge with the existing `manuskript.spec` file.  This makes it harder to troubleshoot issues later on.